### ISSUE TITLE
How to use the CocoVideoDataset?

### ISSUE THREAD

**Issue by lijoe123** [open-mmlab/mmtracking#801]
If i want to use the COCO Dataset, i need to set `load_as_video=False`?, and fill the `CLASSES=(the class name)`,  like the coco dataset name?
![image](https://user-images.githubusercontent.com/86214036/207848402-41188658-1418-47d9-aeae-7dccda91d85e.png)

**Comment by dyhBUPT** [comment]
Yes, you can also directly use the CocoDataset from mmdet.

**Comment by lijoe123** [comment]
Thank you for your answer.

**Comment by eatbreakfast111** [comment]
> Yes, you can also directly use the CocoDataset from mmdet.

Oh, really! I can use the cocodataset from mmdet in mmtracking? Could you please tell me how to do it?
Thank you

**Comment by dyhBUPT** [comment]
> > Yes, you can also directly use the CocoDataset from mmdet.
> 
> Oh, really! I can use the cocodataset from mmdet in mmtracking? Could you please tell me how to do it? Thank you

Yes, but it depends on your model and task. Please refer to #804.

**Comment by eatbreakfast111** [comment]
Which model and task can use it?

**Comment by dyhBUPT** [comment]
If your model doesn't need the reference images, you can use `CocoDataset`, e.g, ByteTrack.

The data config of ByteTrack:
https://github.com/open-mmlab/mmtracking/blob/b1679f990bce7abc18e852004d2f70b09904e238/configs/mot/bytetrack/bytetrack_yolox_x_crowdhuman_mot17-private-half.py#L82-L112

The `forward_train` of ByteTrack:
https://github.com/open-mmlab/mmtracking/blob/b1679f990bce7abc18e852004d2f70b09904e238/mmtrack/models/mot/byte_track.py#L41-L43
